tsdef/types.d.ts

```diff
@@ -190,7 +188,7 @@ export interface ConnectOptions {
   maxVideoBitrate?: number | null;
   name?: string | null;
   networkQuality?: boolean | NetworkQualityConfiguration;
-  notifyWarnings?: Array<TwilioWarning>;
+  notifyWarnings?: Array<'recording-media-lost'>;
```

Comment:
Even though we are not using the term TwilioWarning, having a type to identify possible warnings would be good. How about calling them NotifyWarnings?

```
export type NotifyWarnings = 'recording-media-lost';
```